### DISCUSSION TITLE
Allow disabling email logs in development

### DISCUSSION THREAD

**Discussion by Floriferous** [meteor/meteor#12188] [Feature Requests]
Our Meteor server logs huge emails in the console which is really problematic during our tests (and even in development) as it completely saturates your terminal. Emails with CSS and markup take hundreds of lines for every one sent.

One of our test suite verifies all the ways you can create an account, and therefore logs lots of enrolment emails, email verification, etc.

We'd really like to easily disable these email logs without forking the email package:

```sh
====== END MAIL #1 ======

...email html...

====== BEGIN MAIL #2 ======

```

**Comment by StorytellerCZ** [comment]
I think we can as a first step only show the text version of the emails as that makes more sense in console.
